Packs/Carbon_Black_Enterprise_Response/Integrations/CarbonBlackResponseV2/CarbonBlackResponseV2.yml

```diff
@@ -0,0 +1,2082 @@
+category: Endpoint
+commonfields:
+  id: VMware Carbon Black EDR
+  version: -1
+configuration:
+- display: Server URL
+  name: url
+  required: true
+  type: 0
+- displaypassword: API Token
+  name: credentials
+  type: 9
+  required: false
+  hiddenusername: true
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Filter alerts by query
+  additionalinfo: |-
+    Advanced query string. Accepts the same data as the search box on the Alert Search page.
+    For more information on the query syntax see https://developer.carbonblack.com/resources/query_overview.pdf.
+    If provided, other search filters are not allowed.
+  name: alert_query
+  type: 0
+  required: false
+- display: Filter alerts by status
```

Comment:
any default value?

Reply:
We can but it would be inconvenient to customers that only wants to filer using query, for example. Do we want to do it?

Reply:
either that or have a defaultvalue at `alert_query`. 
so when a customer puts the server and the API key and will click the `fetch incidents` it will just start fetching.

Reply:
@Arsenikr I feel like an input on you side is needed, since we said at least one of the filters input is required for fetch, If the customer will just run it without looking at the incidents, and use a default one, doesn't that misses the point?

Reply:
decided with @Arsenikr  to add default status of unresolved.